playbooks/openshift-node/private/configure_nodes.yml

```diff
@@ -12,6 +12,9 @@
   roles:
   - role: openshift_clock
   - role: openshift_cloud_provider
+    when:
+    - openshift_cloudprovider_kind is defined
+    - openshift_cloudprovider_kind != ''
```

Comment:
openshift_cloudprovider_kind is an inventory variable.  No need to check for '' as that shouldn't be set at all if users aren't defining cloudprovider_kind.

Reply:
Probably could define something like openshift_clouldprovider_known_kind list in the role and check if cloud_provider is in there here instead of ''